build/common.props

```diff
@@ -1,7 +1,7 @@
 <Project>
   <PropertyGroup>
     <!-- Packages can have independent versions and only increment when released -->
-    <Version>3.0.34$(VersionSuffix)</Version>
+    <Version>3.0.35$(VersionSuffix)</Version>
```

Comment:
Should this be a minor version rev as it is a new feature being added?

Reply:
I remember someone saying it was patch but I could be wrong and it should be minor 🤔

@mathewc and @fabiocav for validation

Reply:
Confirmed with Fabio that minor bump works

Reply:
Reverting to patch as suggested by Mathew - we're not changing SDK behaviors and just providing new capabilities for extensions to leverage so patch fits the case fine.